Guided Hunting - Anomalous Office365 Exchange Sessions.ipynb

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
<p>Its often good to have some sort output, even if its just data.head() - i think it helps people who are not so used to Notebooks know that its done something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>